clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/deploy/validators/OpenstackAttributeValidator.groovy

```diff
@@ -102,6 +102,18 @@ class OpenstackAttributeValidator {
     result
   }
 
+  boolean validateHeatTemplate(String value, String attribute, AccountCredentialsProvider accountCredentialsProvider, String account) {
+    def result
+    def credentials = accountCredentialsProvider.getCredentials(account)
+    if (((OpenstackCredentials)credentials.getCredentials()).getProvider().getClient().heat().templates().validateTemplate(value).isValid()) {
+      result = true
```

Comment:
Can we refactor to something like this?
def client = accountCredentialsProvider.getCredentials(account)?.credentials?.provider?.client
if (client && client.heat().templates().validateTemplate(value)?.isValid()) {
....
}